docs/index.rst

```diff
@@ -0,0 +1,174 @@
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: API Reference
+
+   reference/fairstep
+   reference/fairworkflow
+   reference/prov
```

Comment:
I think this is all the user needs, do you agree?

Reply:
Yes, I think so. Between this and the introductory notebook it essentially covers everything.